WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/wizards/RegistrationWizard.java

```diff
@@ -0,0 +1,76 @@
+package nl.tudelft.watchdog.ui.wizards;
+
+import nl.tudelft.watchdog.ui.preferences.Preferences;
+import nl.tudelft.watchdog.ui.util.UIUtils;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectCreatedEndingPage;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectIdEnteredEndingPage;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectRegistrationPage;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectSliderPage;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectWelcomePage;
+
+import org.eclipse.jface.wizard.Wizard;
+
+/** Parent class for User and Project registration wizards. */
+public abstract class RegistrationWizard extends Wizard {
```

Comment:
WelcomePage, IdEnteredEndingPage, RegistrationEndingPage are all base classes for the pages in specific wizards, but they don't have Base in name. Should only this be renamed, leave it like it is, or rename everything?